composer.json

```diff
@@ -17,11 +17,12 @@
     ]
   },
   "minimum-stability": "dev",
+  "prefer-stable": true,
```

Comment:
Using this setting can let this development repository install stable dependencies.